docs/code/use_internal_endpoint.py

```diff
@@ -21,7 +21,7 @@
 gas = GAS("<YOUR_ACCESSKEY>")
 
 # Organize the local dataset by the "Dataset" class before uploading.
-dataset = Dataset("DatasetName")
+dataset = Dataset("<YOUR_DATASETNAME>")
```

Comment:
why some dataset name with underscore: `DATASET_NAME`, some without: `DATASETNAME`?

Reply:
I will fix this problem, just use `YOUR_DATASETNAME`.